app/controllers/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,15 @@
+class ForgotPasswordsController < ApplicationController
+
+
+def create
```

Comment:
This method should be indented, and the blank spaces on either side of it are not necessary.